python/GangaLHCb/Lib/GaudiRunApp/GaudiRunAppUtils.py

```diff
@@ -0,0 +1,78 @@
+import subprocess
+import time
+from Ganga.Core.exceptions import GangaException
+from Ganga.Runtime.GPIexport import exportToGPI
+from Ganga.Utility.logging import getLogger
+from Ganga.Core.exceptions import ApplicationConfigurationError
+from .PythonOptsCmakeParser import PythonOptsCmakeParser
+
+logger = getLogger()
+
+def readInputData(optsfiles, app):
```

Comment:
Introducing two new functions which are line-for line identical (except some small differences which make it more confusing since they imply that they are significant but they are not) is a bad idea. It increases the maintenance load and makes the code harder to understand.